package.json

```diff
@@ -20,7 +20,7 @@
   "author": "The Sails Company",
   "license": "MIT",
   "dependencies": {
-    "debug": "2.2.0",
+    "debug": "^2.6.9",
```

Comment:
@fenichelar In the Sails project, we pin the SVRs of all dependencies that are not maintained by our core team.  Would you mind changing that to `2.6.9`, and also updating https://github.com/sailshq/roadstead/blob/2843d86e075e0cdec34ac8d20b72ad343b88c085/constants/verified-releases.type.js#L34 ?